governance/SIG Security Charter.md

```diff
@@ -1,97 +1,73 @@
+- Provide SPDX (Software Package Data Exchange) scanning tools to detect violations.
 
 ## Cross-cutting Processes
 
 - Disseminating information to other sigs for action / update/ fixing
+- Review and provide feedback on SIG presented features and implementation
 - Provide SPDX Licensing report violations to corresponding SIGs
-- Advise SIGs on security level of new libraries and add to published compliance list
+- Advise SIGs on security level of new libraries 
 - Provide web and docs team with links to campaign information stored in SIG's repository
-
-- Team must provide compliance automation scanning at periodic intervals and point release intervals for each potential issue and report violations to appropriate SIGs.
-
-- Advise Technical steering committee and Governing board of potential theats and unresolved issues.
-
+- Provide consultation for compliance automation scanning at periodic intervals and point release intervals for each potential issue and report violations to appropriate SIGs.
+- Advise Technical steering committee and Governing board of potential threats and unresolved issues.
 - Items that span or require other SIGs or groups and how it relates to this SIG’s responsibilities
 
 ## Out of Scope
 - Not responsible for fixing any detected or reported security issues.
-- Not responsible for enforcing or acting upon any legal compliance issues
+- Not responsible for enforcing or acting upon any legal compliance issues.
 - Not responsible for implementing security or IP detection tools into automation or build chains, but may advise SIGs on where to implement.
-
 - Items that are optional or are not the responsibility of this SIG.
 
-## SIG Links and lists:
+**SIG Links and lists:**
 
-- Joining this SIG
-- Slack/Discord
-- Mailing list
-- Issues/PRs
-- Meeting agenda & Notes
+- Joining this SIG - https://github.com/orgs/o3de/teams/sig-security
+- Slack/Discord - https://discord.com/channels/805939474655346758/816043899477950475 
+- Mailing list - https://lists.o3de.org/g/sig-security
+- Issues/PRs - https://github.com/o3de/sig-security/issues/
+- Meeting agenda & Notes - https://github.com/o3de/sig-security/issues/1 
 
-## Roles and Organization Management
+**Roles and Organization Management**
 
-SIG Docs adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
+SIG Security adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
 
-## Individual Contributors
-Must provide links and examples to security flaws, discoveries, or potential issues.
+**Individual Contributors**
 
 Additional information not found in the sig-governance related to contributors.
 
-## Maintainers
-Must validate and verify that submitted issue or flaw exists and should be accepted into practices.
+**Maintainers**
```

Comment:
Removing the expected role activities makes this weaker, but may be more accurate to the current state of the SIG guidance (I've been unable to attend meetings, and am unaware)

Reply:
I think the intent is to only call things out where this SIG diverges from standard role behaviours. 

I would add that we expect maintainers to be able to evaluate security issues and ensure they are correctly reported with sufficient information to accept. However we can add maintainer expectations once we have issue response runbook in place.

Reply:
Fixed below. Resolving.